client/my-sites/menus/menu-editable-item.jsx

```diff
@@ -24,9 +25,17 @@ var siteMenus = require( 'lib/menu-data' ),
 var Button = React.createClass({
 	render: function() {
 		var className = ( this.props.className || '' ) + ' button';
+    var icon = this.props.icon;
```

Comment:
I don't think this `var` is necessary, we can just use `this.props.icon`, since it's only used in two places.